drivers/lis3mdl/lis3mdl.c

```diff
@@ -120,14 +116,14 @@ void lis3mdl_read_mag(const lis3mdl_t *dev, lis3mdl_3d_data_t *data)
     data->y_axis /= GAUSS_DIVIDER;
     data->z_axis /= GAUSS_DIVIDER;
 
-    i2c_release(dev->i2c);
+    i2c_release(DEV_I2C);
 }
 
 void lis3mdl_read_temp(const lis3mdl_t *dev, int16_t *value)
 {
-    i2c_acquire(dev->i2c);
-    i2c_read_regs(dev->i2c, dev->addr, LIS3MDL_TEMP_OUT_L_REG, (uint8_t*)value, 2);
-    i2c_release(dev->i2c);
+    i2c_acquire(DEV_I2C);
+    i2c_read_regs(DEV_I2C, DEV_ADDR, LIS3MDL_TEMP_OUT_L_REG, (uint8_t*)value, 2);
```

Comment:
is it safe to read 2x 1-byte into a 2-bytes datatype, anyway?

Reply:
not memory but value-wise?

Reply:
I admit, I didn't look at that in detail but it looks valid: *value parameter of `read_temp` function is the address of the 2-bytes datatype variable and (uint8_t*)value cast it to an address of 2-bytes array.